src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 2 && days < 7) {
+    return 40 * days - 20;
```

Comment:
you can create variable for days*40)but in overall good:)

Reply:
Agree, you repeat `days*40` calculation 3 times, create a variable for that